common-extensions/src/main/java/com/twocoders/util/commonextensions/CommonExtensions.kt

```diff
@@ -0,0 +1,26 @@
+package com.twocoders.util.commonextensions
+
+import android.os.Build
+
+fun isLollipopWithAnExtraSugar(): Boolean = Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP_MR1
```

Comment:
tieto mena tiez nie su uplne dobre. Ked sa pytas `isNougat()` chces aby ti to vratilo true pre `Oreo`? Oreo nie je Nughat... Meno nenaznacuje, ze toto kontroluje v podstate `isAtLeastNoughat()`

Reply:
pravdu máš, ale tím že většinou tě nezajímá, že je to přímo == tenAndroid, ale alespoň >= ten Android, tak jsme tam to "AtLeast" nedávali, ale takto public by to tak mělo být, souhlas, doplním ;)